test/server.js

```diff
@@ -1269,6 +1271,93 @@ describe("Server", function () {
         });
     });
 
+    it("should not store the successful verification result when the query parameter dryrun=true is provided", (done) => {
```

Comment:
Does it make sense to group these tests under a nested `describe`, _e.g._, `describe("dryrun=true", [...])`. This way might be easier to filter if needed.

Reply:
Yes this might be handy.